src/external/spglib.jl

```diff
@@ -1,40 +1,45 @@
-using PyCall
 # Routines for interaction with spglib
+# Note: spglib/C uses the row-major convention, thus we need to perform transposes
+#       between julia and spglib.
+#       However, spglib expects the lattice to be passed in as a C Array of the cell vectors,
```

Comment:
I like the term "array of arrays" for this. We don't usually indent like this in comments (but I'm fine with it)

Reply:
Fine from my end.